source/Octopus.Server.Client/Model/WorkerResource.cs

```diff
@@ -35,7 +35,5 @@ public WorkerResource ClearWorkerPools()
             WorkerPoolIds.Clear();
             return this;
         }
-
-        public string SpaceId { get; set; }
```

Comment:
What's the reason we're removing the `SpaceId` from `WorkerResource`?

Reply:
Because it's already inherited from MachineResource.

Reply:
Got it. Makes sense